.drone/drone.yml

```diff
@@ -41,11 +41,6 @@ steps:
   - test
   - lint
 
-- name: shellcheck
```

Comment:
Why was this removed?

Reply:
It was added without being put in jsonnet: https://github.com/grafana/loki/commit/0cde54c4dc6d8ecebe22590771a3fec39d6d2bf5
Will fix

Reply:
Ah thanks! @jeschkies heads up - for future reference, you should modify the `.drone/drone.jsonnet` file instead of the yaml directly :+1:

Reply:
Thanks. Could we add a README.md to `.drone`? I was confused why there are a jsonnet and .yml file. How are new steps added?